cockatrice/src/tab_deck_editor.cpp

```diff
@@ -764,6 +771,19 @@ void TabDeckEditor::actPrintDeck()
     dlg->exec();
 }
 
+void TabDeckEditor::actExportDeckDecklist()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    if(deck != nullptr){
```

Comment:
`if (deck) {`

Reply:
@ZeldaZach We should pick one and set it in the style guide - I just told him to use `nullptr` (which I think is clearer).

Let's leave this as is now. Sorry for the confusion @jrobe